src/libs/TransactionUtils.ts

```diff
@@ -447,7 +447,14 @@ function isReceiptBeingScanned(transaction: OnyxEntry<Transaction>): boolean {
  * Check if the transaction has a non-smartscanning receipt and is missing required fields
  */
 function hasMissingSmartscanFields(transaction: OnyxEntry<Transaction>): boolean {
-    return Boolean(transaction && hasReceipt(transaction) && !isDistanceRequest(transaction) && !isReceiptBeingScanned(transaction) && areRequiredFieldsEmpty(transaction));
+    return Boolean(transaction && !isDistanceRequest(transaction) && !isReceiptBeingScanned(transaction) && areRequiredFieldsEmpty(transaction));
+}
+
+/**
+ * Check if the transaction has missing required fields
+ */
+function hasMissingRequiredFields(transaction: OnyxEntry<Transaction>): boolean {
```

Comment:
I think it is better to just use `hasMissingSmartScanFields` at the places where we added `hasMissingRequiredFields` even though the name `hasMissingSmartScanFields` might be a little misleading for other requests like manual requests with receipts. 

Adding another function with a different name like this `hasMissingRequiredFields` that returns the same value can be more confusing later IMO. @dukenv0307 

cc: @Julesssss

Reply:
I'm on the fence. Having a better-described function name is good. But we can always do that later, or rename the existing function. @dukenv0307 is the idea that we'll eventually add to these functions to separate them?